collections/_publications/2018-Rueth-ECCMECFD.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Time stepping algorithms for partitioned multi-scale multi-physics in preCICE"
-pub-url: http://www.eccm-ecfd2018.org/admin/files/filePaper/p1640.pdf
+pub-url: http://congress.cimne.com/eccm_ecfd2018/admin/files/filePaper/p1640.pdf
```

Comment:
Note to all interested: ECCM-ECFD2018 moved their website and abstracts. Check if you have anything linking to that.